src/functions.ts

```diff
@@ -42,8 +45,16 @@ type LanguageType = "en" | "es" | "fr" | "tr";
 
 function printHello(name: string, language: LanguageType): string {
   // write your code here
-
-  return ""; // replace the empty string with what you see is fit
+  switch (language) {
+    case "en":
+      return `Hello ${name}`;
+    case "es":
+      return `Hola ${name}`;
+    case "fr":
+      return `Bonjour ${name}`;
+    case "tr":
+      return `Merhaba ${name}`;
+  } // replace the empty string with what you see is fit
```

Comment:
Something that I personally prefer to switch statements is to use an object literal as a "dictionary":

```TS
const greetings = {
  en: "Hello",
  es: "Hola",
  fr: "Bonjour",
  tr: "Merhaba",
};

return `${greetings[language]} ${name}`
```